src/requirements/Twitter/components/TwitterVerificationSelect.tsx

```diff
@@ -0,0 +1,54 @@
+import { FormControl, FormLabel } from "@chakra-ui/react"
+import ControlledSelect from "components/common/ControlledSelect"
+import { useFormContext } from "react-hook-form"
+import { RequirementFormProps } from "requirements"
+import { SelectOption } from "types"
+import parseFromObject from "utils/parseFromObject"
+
+type ValueType = "any" | "blue" | "business" | "government"
+
+const options: SelectOption<ValueType>[] = [
+  {
+    label: "One of them",
+    value: "any",
+  },
+  {
+    label: "Blue",
+    value: "blue",
+  },
+  {
+    label: "Business",
+    value: "business",
+  },
+  {
+    label: "Government",
+    value: "government",
+  },
+]
+
+const TwitterVerificationSelect = ({
```

Comment:
Since we use this component only for 1 requirement, let's mirror the requirement's name here and rename it to `TwitterAccountVerified`